tex/latex/biblatex/biblatex.def

```diff
@@ -2803,6 +2804,26 @@
 
 \newbibmacro*{date}{\printdate}
 
+\DeclareFieldFormat{legaldates}{\mkbibparens{#1}}
+\newbibmacro*{legaldates}{
```

Comment:
The line is still missing a `%` to protect the line end.
```suggestion
\newbibmacro*{legaldates}{%
```
A few protecting `%` in the following lines are unnecessary. Mostly the ones in the first argument of `\ifboolexpr`, but also the one `{}%` in l. 2814 and the one after `\newunit`.

Reply:
ok